data/campaigns/Heir_To_The_Throne/scenarios/19c_Cliffs_of_Thoria.cfg

```diff
@@ -716,7 +716,7 @@ Soooo... It is you who sent your subordinates to attack us. Now when we’ve des
         [message]
             speaker=unit
             scroll=no
-            message= _ "It is a young human sergeant, barely more than a boy. He appears badly hurt but still alive."
+            message= _ "It is a young human officer, barely more than a boy. He appears badly hurt but still alive."
```

Comment:
There seems to be a wordplay related to the name of Warven being similar to Dwarven, and that dwarves _may_ be confused with human children. Is it worth adding a #po comment? Or might I be overthinking it?

Reply:
Well I don't think it's a problem for me reading it since it's clearly in a name field?